src/components/FocusButton.jsx

```diff
@@ -30,7 +30,7 @@ const FocusButton = (props) => {
   const focused = focusData.store === store && focusData.key === lookupKey;
 
   return (
-    <Tooltip placement='right' title={`${(focused) ? 'Hide color on' : 'Show color on'} embedding`}>
+    <Tooltip placement='right' title={`${(focused) ? 'Hide color on' : 'Show color on'} embedding`} destroyTooltipOnHide>
```

Comment:
I assume that you added `destroyTooltipOnHide` to improve the performance? Is this something that we need to be adding for every tooltip?

Reply:
No, this was in order to prevent the tooltip "sticking" to the button when we hide/show cell sets.